File: app/src/main/java/com/paycraft/ticketbooker/models/OTP.java
package com.paycraft.ticketbooker.models;

import com.google.gson.annotations.SerializedName;


public class OTP {

    @SerializedName("otp")
    private
    int otp;

    public OTP(int otp) {
        this.otp = otp;
    }

    public int getOtp() {
        return otp;
    }

    public void setOtp(int otp) {
        this.otp = otp;
    }

    @Override
    public String toString() {
        return "OTP{" +
                "otp=" + otp +
                '}';
    }
}

File: app/src/main/java/com/paycraft/ticketbooker/models/OTP.kt
package com.paycraft.ticketbooker.models

import com.google.gson.annotations.SerializedName


class OTP(@field:SerializedName("otp")
          var otp: Int) {

    override fun toString(): String {
        return "OTP{" +
                "otp=" + otp +
                '}'.toString()
    }
}
